### PR TITLE
Sever backend laces

### PR DESCRIPTION
**Pull request by FluffynatorT800**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
+require('dotenv').config();
 const express = require("express");
 const mongoose = require("mongoose");
 const app = express();
+
+const shoeRoutes = require("./routes/laces.routes");
```
Comment (elenaserezhkina) [inline server.js]: Why are you importing shoe routes from file laces routes?
And why do you have two const variables with the same name?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-node_modules
+.env
+/node_modules/
```
Comment (elenaserezhkina) [inline .gitignore:2]: please add also `.vs` and `.vscode`  to gitignore

Comment (elenaserezhkina) [reply .gitignore:2]: and also remove the vs nd vscode files from the commit

File: test/fixtures/shoe.fixture.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+const shoeTwo = {
+    _id: mongoose.Types.ObjectId(),
+    brand: "Converse",
+    colorVariant: [{
+        color: "red",
+        image:{ binData: 'test' }
+    }],
+    laceImg: { binData: 'test' }
+};
+const insertShoes = async users => {
```
Comment (elenaserezhkina) [inline test/fixtures/shoe.fixture.js]: what are users here?

**Review by elenaserezhkina — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (FluffynatorT800) [reply .gitignore:2]: okay done

File: test/fixtures/shoe.fixture.js (hunk 3 above)
Comment (FluffynatorT800) [reply test/fixtures/shoe.fixture.js]: just a placeholder I replaced it with shoesers

**Pull request closed without merging**